.github/ISSUE_TEMPLATE/invitation.yml

```diff
@@ -24,9 +25,9 @@ body:
       label: Additional Context
       description: Where did you learn about Learn With Community? What do you like about this community/Why do you want to join?
     validations:
-      required: false
+      required: true
```

Comment:
I don't think additional context should be required

Reply:
> I don't think additional context should be required

It is required in EddieHub and in other communities as well, so that's why I thought of enabling it.

Reply:
@Panquesito7 In EddieHub you suggested this, EddieHub is a huge community, we have just been introduced 🙂